lib/src/views/playing-card-content-view.dart

```diff
@@ -31,59 +31,51 @@ class PlayingCardContentView extends StatelessWidget {
         double innerWidth = width * 1.6875 / 2.5;
         double innerHeight = height * 2.8125 / 3.5;
         double sideSpace = (width - innerWidth) / 2.0;
-        double labelHeight = height * 0.089285;
         double suitHeight = height * 0.160714;
         double labelSuitHeight = suitHeight / 2.0;
+        double topOffset = height * 0.030714;
 
         TextStyle ts = valueTextStyle!.copyWith(
-            fontSize: getGoodFontSize("10", valueTextStyle!, sideSpace * .9));
+            fontSize: getGoodFontSize("I0", valueTextStyle!, sideSpace * .9,
```

Comment:
Perhaps the label should be overrideable in the style. I imagine some function that recieves a CardValue and outputs "10" or "I0" or whatever the user desires. Ideally, this would have reasonable defaults. This could happen in a future change.